layouts/partials/fragments/contact.html

```diff
@@ -142,8 +142,9 @@ <h2>{{- . | markdownify -}}</h2>
                     </button>
                     <span class="filler"></span>
                   </div>
+                  <div data-netlify-recaptcha></div>
                 {{- else }}
-                  <button
+                    <button
```

Comment:
Should there be extra whitespace here?

Reply:
Yeah, that's a formatting error. I corrected in the second PR.